references/identity-facade/README.md

```diff
@@ -24,7 +24,7 @@ connect to the IdP
 
 ## Dependencies
 
-- [Maven](https://maven.apache.org/)
+- [Maven (Version >= 3.6.1)](https://maven.apache.org/)
```

Comment:
aim for 3.6.0 if possible for compatibility with debian repos

Reply:
ACK removed ntp from sackmesser for now